### PR TITLE
Feature/ai door electricute for real

### PR DESCRIPTION
**Pull request by Fildrance**
## Requirements

- [ ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Shared/Doors/Systems/SharedDoorSystem.Bolts.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ public void SetBoltLightsEnabled(Entity<DoorBoltComponent> ent, bool value)
 
     public void SetBoltsDown(Entity<DoorBoltComponent> ent, bool value, EntityUid? user = null, bool predicted = false)
     {
-        if (ent.Comp.BoltsDown == value)
+        if (ent.Comp.BoltsDown == value || !ent.Comp.Powered)
```
Comment (Fildrance) [inline Content.Shared/Doors/Systems/SharedDoorSystem.Bolts.cs:80]: looks SUS

File: Content.Client/Silicons/StationAi/StationAiSystem.Airlock.cs [hunk 2]
```diff
@@ -9,6 +10,8 @@ public sealed partial class StationAiSystem
     private void InitializeAirlock()
     {
         SubscribeLocalEvent<DoorBoltComponent, GetStationAiRadialEvent>(OnDoorBoltGetRadial);
+        SubscribeLocalEvent<AirlockComponent, GetStationAiRadialEvent>(OnEmergencyAccessGetRadial);
+        SubscribeLocalEvent<ElectrifiedComponent, GetStationAiRadialEvent>(OnDoorElectrifiedGetRadial);
```
Comment (Fildrance) [inline Content.Client/Silicons/StationAi/StationAiSystem.Airlock.cs:14]: that could be a problem

File: Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs [hunk 3]
```diff
@@ -90,22 +90,22 @@ private void AddTricksVerbs(GetVerbsEvent<Verb> args)
                 args.Verbs.Add(bolt);
             }
 
-            if (TryComp<AirlockComponent>(args.Target, out var airlock))
+            if (TryComp<AirlockComponent>(args.Target, out var airlockComp))
```
Comment (Fildrance) [inline Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs:93]: most likely gonna get reverted

File: Content.Server/Remotes/DoorRemoteSystem.cs [hunk 4]
```diff
@@ -74,7 +74,7 @@ private void OnBeforeInteract(Entity<DoorRemoteComponent> entity, ref BeforeRang
                 case OperatingMode.ToggleEmergencyAccess:
                     if (airlockComp != null)
                     {
-                        _airlock.ToggleEmergencyAccess(args.Target.Value, airlockComp);
+                        _airlock.ToggleEmergencyAccess((args.Target.Value, airlockComp));
```
Comment (Fildrance) [inline Content.Server/Remotes/DoorRemoteSystem.cs:77]: revert?..

File: Content.Shared/Electrocution/Components/ElectrifiedComponent.cs [hunk 5]
```diff
@@ -1,121 +1,125 @@
...
-    [DataField("shockVolume")]
+    [DataField("shockVolume"), AutoNetworkedField]
     public float ShockVolume = 20;
 
-    [DataField]
+    [DataField, AutoNetworkedField]
     public float Probability = 1f;
+
+    [DataField, AutoNetworkedField]
+    public bool IsWireCut = false;
```
Comment (Fildrance) [inline Content.Shared/Electrocution/Components/ElectrifiedComponent.cs:124]: made this work the same way DoorRemote is checking Bolts - DoorBoltComponent have same property and it is changed in the same way.

File: Content.Shared/Electrocution/Components/ElectrifiedComponent.cs [hunk 6]
```diff
@@ -1,121 +1,125 @@
...
 
-namespace Content.Server.Electrocution;
+namespace Content.Shared.Electrocution;
 
 /// <summary>
 ///     Component for things that shock users on touch.
 /// </summary>
-[RegisterComponent]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
 public sealed partial class ElectrifiedComponent : Component
```
Comment (Fildrance) [inline Content.Shared/Electrocution/Components/ElectrifiedComponent.cs:10]: Moved to shared, marked as networked, no system code was moved to shared tho. It looks ok but i was strongly discouraged from doing this... I'd be very happy if i had idea on 'how' - moving system and making predictable ElectrocutionSystem is by all means no easy task

File: Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs (hunk 3 above)
Comment (ScarKy0) [reply Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs:93]: I was told to do it

**Pull request closed without merging**